example/spec/parallel/discoverGranules/DiscoverGranulesDuplicateHandlingSkipSpec.js

```diff
@@ -197,10 +197,16 @@ describe('The DiscoverGranules workflow with one existing granule, one new granu
 
     await pAll(
       [
-        () => deleteS3Object(sourceBucket, existingGranuleKey),
-        () => deleteS3Object(sourceBucket, newGranuleKey),
         () => deleteGranule({ prefix, granuleId: existingGranuleId }),
         () => deleteGranule({ prefix, granuleId: newGranuleId }),
+      ],
+      { stopOnError: false }
```

Comment:
@npauzenga noting as this is one of the :thumbsup: patterns I was talking about instead of the Promise.all setups a lot of the tests are using.